scaleway/resource_k8s_cluster_test.go

```diff
@@ -664,6 +664,7 @@ func testAccCheckScalewayK8SClusterConfigPrivateNetworkNotLinked(version string)
 	return fmt.Sprintf(`
 resource "scaleway_vpc_private_network" "private_network" {
   name       = "k8s-private-network"
+  is_regional = true
```

Comment:
I don't think this field will stay once vpc v1 is deprecated. Is there a way to manage it transparently for the customer?

Reply:
It is transparent for the customer because both v1 and v2 work for the moment, so they can choose if they want to add `is_regional = true` to use v2 or keep using v1.
The goal of this PR is to set up everything for v2 while still supporting v1, that's why I updated the test so that line 536 uses the regional function rather than the zoned one.